README.md

```diff
@@ -0,0 +1,60 @@
+# O3DE multi-robot demo
+
+## Goal
+
+This project serves the purpose to demonstrate [O3DE](https://www.o3de.org/) and its simulation capabilities [Interactivity and simulation](https://www.docs.o3de.org/docs/user-guide/interactivity/).
+This project demonstrates an example of application complex cooperative ROS 2 application.
+The integration is realized through [ROS 2 Gem](https://github.com/o3de/o3de-extras/blob/development/Gems/ROS2).
+
+## How does it look like
+<img src="media/view1.png" width="640">
+<img src="media/view2.png" width="640">
+
+## The project includes
+- **Scenery** imported created using a [Warehouse project template](https://www.docs.o3de.org/docs/user-guide/interactivity/robotics/project-configuration/#ros-2-project-templates)
+- **Robotic Arms** imported using [URDF description](https://github.com/UniversalRobots/Universal_Robots_ROS2_Description) provided by Universal Robotics for their UR10 cobot .  
+- **AMRs** instatinated prefabs of Proteus AMR robot from [ProteusRobot Gem](https://github.com/o3de/o3de-extras/tree/development/Gems/ProteusRobot)
+- **Boxes** that are transported using conveyor belts and are to be paletized
+
+## Platforms
+
+The project runs on Ubuntu 22.04 with ROS 2 Humble or ROS 2 Iron.
+
+💡 ***Note:*** This demo is **not supported on Windows!** 
+
+## O3DE
+
+1. Refer to the [O3DE System Requirements](https://www.o3de.org/docs/welcome-guide/requirements/) documentation to make
+   sure that the system/hardware requirements are met.
+2. Please follow the instructions
+   to [set up O3DE from GitHub](https://o3de.org/docs/welcome-guide/setup/setup-from-github/).
+3. **Use the `development` branch**.
+
+The following commands should prepare O3DE:
+
+```
+~$ git clone --branch development --single-branch https://github.com/o3de/o3de.git
+~$ cd o3de
+~/o3de$ git lfs install
+~/o3de$ git lfs pull
+~/o3de$ python/get_python.sh
+~/o3de$ scripts/o3de.sh register --this-engine
+```
+
+## ROS 2 Gem
+
+This project uses the [ROS 2 Gem](https://github.com/o3de/o3de-extras/blob/development/Gems/ROS2).
+Please make sure to follow the installation guide
```

Comment:
While this instruction is very useful and complete, it might be quite confusing for newbies - there is a section about creating a project from a template, but it might be not clear what to do with this repo (no need for a template, this repo is a project itself). If we want to make this demo as easy as possible, we should keep this in mind and redo some docs.